src/context/CardInfoContext.tsx

```diff
@@ -0,0 +1,58 @@
+import { createContext, useState } from 'react';
+import { Card } from 'components/common/Card/types';
+
+export type CardInfoContextType = {
+  cardInfo: Card;
+  setCardInfo: (card: Card) => void;
+  setCardInfoName: (cardName: string) => void;
+};
+
+export interface ContextProps {
+  children: JSX.Element | JSX.Element[];
+}
+
+export const defaultCardInfo: Card = {
+  bank: '현대카드',
+  numbers: ['', '', '', ''],
+  expirationDate: { month: '', year: '' },
+  ownerName: '',
+  securityCode: '',
+  password: {
+    first: '',
+    second: '',
+  },
+};
+
+export const CardInfoContext = createContext<CardInfoContextType>({
+  cardInfo: defaultCardInfo,
+  setCardInfo: () => {},
+  setCardInfoName: () => {},
+});
+
+export const CardInfoProvider = ({ children }: ContextProps) => {
+  const [cardInfo, setCardInfo] = useState<Card>(defaultCardInfo);
+
+  const updateCard = (card: Card) => {
+    setCardInfo((prev) => ({
+      ...prev,
+      bank: card.bank,
+      numbers: card.numbers,
+      expirationDate: card.expirationDate,
+      ownerName: card.ownerName,
+      securityCode: card.securityCode,
+      password: card.password,
+    }));
```

Comment:
데이터의 키와 값이 동일하게 들어온다면 아래와 같이 수정해도 되겠네요 ㅎㅎ

```suggestion
    setCardInfo((prev) => ({
      ...prev,
      ...card,
    }));
```

Reply:
그렇네요..! 해당 부분 피드백 반영하였습니다😭